### PR TITLE
Removes polychrome

### PR DESCRIPTION
**Pull request by tf-4**
## About The Pull Request

Removes `/datum/element/polychromic` and all references to polychromism

## How This Contributes To The Skyrat Roleplay Experience

It's finally dead. GAGS has replaced it, and now it can be phased out. RIP, you served us well-ish.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
</details>

## Changelog

Not player-facing

### CODE REVIEW

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_suit.dm [hunk 1]
```diff
@@ -356,7 +356,7 @@ GLOBAL_LIST_INIT(loadout_exosuits, generate_loadout_items(/datum/loadout_item/su
 	item_path = /obj/item/clothing/suit/fallsparka
 
 /datum/loadout_item/suit/poly
-	name = "Polychromic Wintercoat"
+	name = "Colourable Wintercoat"
```
Comment (ghost) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_suit.dm]: Best call it recolorable to fit with the other gags loadout items

Also maybe needs to be repathed from /poly but that might mean people have to readd it to loadout

Either way good riddance to poly

**Review by ghost — COMMENTED** [review]
All the homie hate poly

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_suit.dm (hunk 1 above)
Comment (tf-4) [reply modular_skyrat/modules/loadouts/loadout_items/loadout_datum_suit.dm]: True but may as well do it now

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_suit.dm [hunk 2]
```diff
@@ -356,7 +356,7 @@ GLOBAL_LIST_INIT(loadout_exosuits, generate_loadout_items(/datum/loadout_item/su
 	item_path = /obj/item/clothing/suit/fallsparka
 
 /datum/loadout_item/suit/poly
```
Comment (tf-4) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_suit.dm]:
```suggestion
/datum/loadout_item/suit/gags_wintercoat
```

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_suit.dm (hunk 1 above)
Comment (tf-4) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_suit.dm]:
```suggestion
	name = "Recolorable Winter Coat"
```

**Review by tf-4 — COMMENTED** [review]
Verified ☑

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_suit.dm (hunk 1 above)
Comment (GoldenAlpharex) [reply modular_skyrat/modules/loadouts/loadout_items/loadout_datum_suit.dm]: Also please, `Colorable` not `Colourable`.

**Review by Zonespace27 — APPROVED** [review]
Rest in shit

**Review by GoldenAlpharex — APPROVED** [review]
Bye bye!

**Pull request merged**